libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -145,6 +145,9 @@ class AC_WPNav
     /// get_bearing_to_destination - get bearing to next waypoint in centi-degrees
     virtual int32_t get_wp_bearing_to_destination_cd() const;
 
+    /// get_bearing_to_destination - get bearing to next waypoint in radians
+    virtual float get_bearing_to_target_rad() const;
```

Comment:
do we really want to use the word "target" here instead of "destination"?  I'm not sure we should introduce the word "target" should we?